android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -192,7 +221,7 @@ public void onPrepareOptionsMenu(Menu menu) {
   @Override
   public void onListItemClick(ListView l, View v, int position, long id) {
     // If we are scanning
-    if (mIsScanRunning) {
+    if (mScanningAnimationDrawable.isRunning()) {
```

Comment:
It's possible that the animation may not be animating, but scanning is still happening. This might not matter after the first timeout, since new items are appended to the list, so perhaps this is moot?